CPP_STYLE_GUIDE.md

```diff
@@ -209,6 +217,24 @@ Never use `std::auto_ptr`. Instead, use `std::unique_ptr`.
 - Use `static_cast` for casting whenever it works
 - `reinterpret_cast` is okay if `static_cast` is not appropriate
 
+### Using `auto`
+
+Being explicit about types is usually preferred over using `auto`.
+
+Use `auto` to avoid type names that are noisy, obvious, or unimportant. When
+doing so, in mind that explicit types often help with readability and verifying
```

Comment:
Nit: those explicit types?

Reply:
I didn’t mentioned `explicit types` as a term before, so this might be a bit confusing? Not sure though

Reply:
The word `keep` seems to be missing.

Reply:
@tniessen Thanks, done!